Gemfile

```diff
@@ -48,6 +49,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  #gem 'rspec-rails'
```

Comment:
is there any reason to comment rspec gem?

Reply:
i'll just start with Rspec